kml/create-and-save-kml-file/README.md

```diff
@@ -0,0 +1,48 @@
+# Create and save KML file
+
+Construct a KML document and save it as a KMZ file.
+
+![Create And Save KML File Sample](CreateAndSaveKMLFile.png)
```

Comment:
we should probably remove this description for now with the sample viewer in mind: same for the other PRs I meant to add.